iOS/UI/Swipeable Input/SwipeInputPanGestureHandler.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  SwipeInputPanGestureRecognizer.swift
+//  Jibber
+//
+//  Created by Martin Young on 1/26/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import Lottie
+
+class SwipeInputPanGestureHandler {
```

Comment:
The swipe accessory view was getting huge. I moved the pan logic into a separate class.